go/mysql/flavor_mysqlgr.go

```diff
@@ -141,32 +141,32 @@ func (mysqlGRFlavor) status(c *Conn) (ReplicationStatus, error) {
 		return res, nil
 	}
 
-	// Populate IOThreadRunning from replication_connection_status
+	// Populate IOState from replication_connection_status
 	query = fmt.Sprintf(`SELECT SERVICE_STATE
 		FROM performance_schema.replication_connection_status
 		WHERE CHANNEL_NAME='%s'`, chanel)
-	var ioThreadRunning bool
+	var connectionState ReplicationState
```

Comment:
the default will be `ReplicationStateRunning`

Reply:
Good catch! 😄 Will change the ordering so that 0 is unknown as you noted. Thanks!